graphbolt/src/fused_csc_sampling_graph.cc

```diff
@@ -663,18 +977,22 @@ c10::intrusive_ptr<FusedSampledSubgraph> FusedCSCSamplingGraph::SampleNeighbors(
     }
   }
 
+  bool with_seed_offsets = seed_offsets.has_value();
```

Comment:
We don't need this boolean if we use torch::optional for the seed offset variables.